src/db/model/stardust/block/output/unlock_condition.rs

```diff
@@ -137,11 +140,18 @@ pub(crate) mod test {
         let bson = to_bson(&block).unwrap();
         assert_eq!(block, from_bson::<UnlockCondition>(bson).unwrap());
 
-        let block = get_test_immut_alias_address_condition(bee_test::rand::output::rand_alias_id().into());
+        let block = get_test_immut_alias_address_condition(get_test_alias_address_as_address());
         let bson = to_bson(&block).unwrap();
         assert_eq!(block, from_bson::<UnlockCondition>(bson).unwrap());
     }
 
+    pub(crate) fn get_test_alias_address_as_address() -> Address {
+        bee_block_stardust::address::Address::from(bee_block_stardust::address::AliasAddress::new(
+            bee_test::rand::output::rand_alias_id(),
+        ))
+        .into()
```

Comment:
Would be a bit more readable if those types were imported (maybe even just within this function with `bee_block_stardust::address::*`).